config.prod.php

```diff
@@ -27,6 +27,12 @@
     # Secret key used for creating user activation hashes
     # 'user-activation-secret' => '',
 
+    # Whether or not to use a special download server. Useful if large downloads are often aborted due to network errors.
+    # 'use_download_server' => true,
```

Comment:
why use a boolean flag? I guess it is enough when there is a setting !== null for `download_server_ip`

Reply:
What if you want to disable the download server temporarily? I think it makes no sense to determine whether or not the download server will be used by the IP that is set here. Having a flag allows for more control.

Reply:
Ok